Dockerfile.quick

```diff
@@ -7,4 +7,6 @@ FROM timescaledev/promscale-extension:${EXTENSION_VERSION}-${TIMESCALEDB_VERSION
 
 ARG EXTENSION_VERSION
 COPY sql/promscale-${EXTENSION_VERSION}.sql /usr/local/share/postgresql/extension/promscale--${EXTENSION_VERSION}.sql
+# TODO (james): This probably needs to be extended to be created for all `upgradeable_from` in promscale.control
```

Comment:
Is there an issue with using `create-upgrade-symlinks.sh` here?

Reply:
Ya, it wasn't designed to be used here and there are enough differences between where it _was_ designed to work and here that I'm not sure if it makes sense to make it work here. As I mention below, I suspect that some of this stuff will change in the near future, so I'd prefer to punt on perfecting that for now.